docs/getting-started.md

```diff
@@ -442,47 +444,40 @@ automatically save the edits to files that you make.**
 ### Add an `age` column
 
 1. The "Explorer" on the left hand side lists the files and folders in
-   your research repository. Find and click on the `study_definition.py`
-   file inside the `analysis` folder. This file specifies the population
-   that you'd like to study (dataset rows) and what you need to know
-   about them (dataset columns).
+   your research repository. Find and click on the `dataset_definition.py`
+   file inside the `analysis` folder. This file contains a dataset definition,
+   specifying the population that you'd like to study (dataset rows)
+   and what you need to know about them (dataset columns).
+   It is written in [ehrQL](/ehrql/).
 1. Add some text so that the file looks like this (new text highlighted):
-```python linenums="1" hl_lines="14 15 16 17 18 19 20"
-from cohortextractor import StudyDefinition, patients, codelist, codelist_from_csv  # NOQA
-
-
-study = StudyDefinition(
-    default_expectations={
-        "date": {"earliest": "1900-01-01", "latest": "today"},
-        "rate": "uniform",
-        "incidence": 0.5,
-    },
-    population=patients.registered_with_one_practice_between(
-        "2019-02-01", "2020-02-01"
-    ),
-
-    age=patients.age_as_of(
-        "2019-09-01",
-        return_expectations={
-            "rate": "universal",
-            "int": {"distribution": "population_ages"},
-        },
-    ),
-)
+```python linenums="1" hl_lines="15"
+from ehrql import Dataset
+from ehrql.tables.beta.tpp import patients, practice_registrations
+
+dataset = Dataset()
```

Comment:
suggestion: Probably better to do this as we instruct elsewhere:

```suggestion
from ehrql import create_dataset
from ehrql.tables.beta.tpp import patients, practice_registrations

dataset = create_dataset()
```

Reply:
Ah yes, good point.  I'll update the template too.